### PR TITLE
[release/9.0-staging] Automate dl 9 0

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

**Review by carlossanlop — CHANGES_REQUESTED** [review]
@lewing - Please retarget this PR to the [release/9.0-staging](https://github.com/dotnet/runtime/tree/release/9.0-staging) branch.

You can retarget to that branch by clicking on the Edit button on the top right (next to the PR title) and choosing the release/9.0-staging branch from the dropdown. Important: Please make sure you don't bring any unrelated changes from the release/9.0 branch when retargeting to release/9.0-staging.

File: eng/Versions.props [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>9.0.100</SdkBandVersion>
-    <PackageVersionNet8>8.0.11</PackageVersionNet8>
+    <PackageVersionNet8>8.0.$([MSBuild]::Add($([System.Version]::Parse('$(ProductVersion)').Build),11))</PackageVersionNet8>
```
Comment (marcpopMSFT) [inline eng/Versions.props:10]: I think you want PatchVersion here and not ProductVersion

Comment (lewing) [reply eng/Versions.props:10]: It parses the ProductVersion, at lest in 8.0 that also changed?

Comment (marcpopMSFT) [reply eng/Versions.props:10]: ahh, I just saw the productversion and didn't notice the parse. Does parsing that and grabbing build do the right thing? I actually don't know.

**Comment by carlossanlop** [comment]
We'll merge the other one instead of this one: https://github.com/dotnet/runtime/pull/110586

**Pull request closed without merging**